### PR TITLE
Task/tam 3/feed the pet

### PR DESCRIPTION
**Pull request by kalibria**

### CODE REVIEW

File: src/mainField/MainField.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+import { StateIndicator } from "../indicators/StateIndicator";
+import { FoodIndicator } from "../indicators/FoodIndicator";
+import { AgeIndicator } from "../indicators/AgeIndicator";
+import ButtonFeed from "./ButtonFeed";
+
+export const MainField = ({ state, indicators }) => {
+  const [foodPercent, setFoodPercent] = useState(state.initialFoodPercent);
+  const [age, setAge] = useState(state.initialAge);
+  const [buttonFeedEntered, setButtonFeedEntered] = useState(false);
```
Comment (ikushlianski) [inline src/mainField/MainField.js:12]: maybe let's call it `buttonFeedEnabled`?

File: src/mainField/ButtonFeed.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+const ButtonFeed = ({ setFoodPercent, setAge, age, setButtonFeedEntered }) => {
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    setButtonFeedEntered(true);
+    setFoodPercent(100);
```
Comment (ikushlianski) [inline src/mainField/ButtonFeed.js]: better not use "magic numbers", instead consider using constants like `MAX_FOOD_PERCENT`

File: src/config.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
+export const config = {
+  state: {
+    name: "",
+    initialStatus: "fine",
+    initialFoodPercent: "100",
```
Comment (ikushlianski) [inline src/config.js:5]: why is it a string? could be a number, because it's percent

**Review by ikushlianski — APPROVED** [review]

File: src/mainField/MainField.js (hunk 1 above)
Comment (kalibria) [reply src/mainField/MainField.js:12]: renamed

File: src/mainField/ButtonFeed.js (hunk 2 above)
Comment (kalibria) [reply src/mainField/ButtonFeed.js]: use initialFoodPercent instead "magic numbers"

File: src/config.js (hunk 3 above)
Comment (kalibria) [reply src/config.js:5]: fixed

**Review by ikushlianski — APPROVED** [review]

**Pull request merged**